vite.config.ts

```diff
@@ -23,33 +25,26 @@ export default defineConfig({
   plugins: [
     crx({ manifest }),
 
-    vue(),
-
-    VueDevTools(),
-
-    Pages({
-      dirs: [
-        {
-          dir: 'src/pages',
-          baseRoute: '',
-        },
-        {
-          dir: 'src/options/pages',
-          baseRoute: 'options',
-        },
-        {
-          dir: 'src/popup/pages',
-          baseRoute: 'popup',
-        },
-        {
-          dir: 'src/content-script/iframe/pages',
-          baseRoute: 'iframe',
-        },
+    VueRouter({
+      root: '.',
+      // Add your own custom pages here. Just add it to the array. Example: 'src/welcome/pages'
+      routesFolder: [
+        'src/pages',
+        'src/popup/pages',
+        'src/options/pages',
+        'src/content-script/iframe/pages',
+        'src/setup/pages',
       ],
+      dts: 'src/typed-router.d.ts',
+      extensions: ['.vue'],
     }),
 
+    vue(),
+
+    // VueDevTools(),
```

Comment:
Any specific reason why we have disabled `devtools` here?

Reply:
> Any specific reason why we have disabled `devtools` here?

It's broken and looks like this: 
![image](https://github.com/mubaidr/vite-vue3-chrome-extension-v3/assets/151234273/5222fb8c-fe74-40e4-87a5-cab47e7270dd)

Reply:
Yeah. right, we need to fix this issue.